tests/rkt_run_pod_manifest_test.go

```diff
@@ -107,6 +107,27 @@ func TestPodManifest(t *testing.T) {
 		cgroup         string
 	}{
 		{
+			// Special characters
+			[]imagePatch{
+				{"rkt-test-run-pod-manifest-read.aci", []string{}},
```

Comment:
Can you give it another name? "rkt-test-run-pod-manifest-special-characters.aci"